ote/src/cljs/ote/app/routes.cljs

```diff
@@ -131,7 +131,7 @@
                      app (merge app navigation-data)]
 
                  (if (requires-authentication? app)
-                   (do (navigate! :front-page)
+                   (do (navigate! :login)
```

Comment:
Login prompt fix included intentionally?

Reply:
yes because it's such a simple change. It could be separated into it's own pr.